### ISSUE TITLE
upgrading the Marlin renderer integrated into Zulu8 (0.7.4 vs latest 0.9.1)

### ISSUE THREAD

**Issue by bourgesl** [zulu-openjdk/zulu-openjdk#17]
Hi
I would like upgrading the Marlin renderer integrated into Zulu8 (0.7.4 vs latest 0.9.1)
I submitted a PR to jetbrains jdk8u_jdk and could make the same for zulu8.
See openjdk (8) branch:
https://github.com/bourgesl/marlin-renderer/tree/openjdk?files=1
If you would have zulu source code @ github, I could submit a pull request

Laurent

**Comment by sgrinev** [comment]
Hi Laurent,
thanks for heads up about new version! We'll update Marlin in Zulu and will include into one of our next releases once it passed testing.

Unfortunately Zulu sources are not on github, see Zulu's readme.txt for the sources request procedure.

**Comment by bourgesl** [comment]
Hi,
If you have your own graphics test suite (regression, benchmark), I am interested to learn more.

Please tell me once you have upgraded Zulu code base, to let me test it.

Will you use the openjdk (8) branch (github) or backport Marlin from jdk/client forrest (jdk 11) after its integration (review started) ?

**Comment by sgrinev** [comment]
Hi Laurent,

for rendering testing we are mostly reusing OpenJDK regression tests which are right in the repository.
Not sure when we will start updating Zulu code, got too much stuff in our hands right now. Hopefully before April CPU.

About best way to get new Marlin we are not yet decided, what would you advice?

**Comment by bourgesl** [comment]
Just ping me when you are ready, so I can ensure the openjdk (8) branch (github) is in synch with openjdk 11 once the RFR is done

**Comment by bourgesl** [comment]
Are you ready ?
Jetbrains JRE8 has integrated Marlin 0.9.3

**Comment by bourgesl** [comment]
Please notify me soon or I will give up

**Comment by sgrinev** [comment]
@bourgesl  Hi Laurent, sorry for the delay, we are still reviewing changes and trying to fit merge into a schedule.

**Comment by bourgesl** [comment]
Please use latest release 0.9.3:
https://github.com/bourgesl/marlin-renderer/commit/0d2f2fe786d80a94400db8c7a2e62dea0f1511a8

**Comment by bourgesl** [comment]
If you share the zulu 8 code (at least your openjdk8 fork) on github, I will propose one Pull Request to upgrade to Marlin 0.9.3

**Comment by bourgesl** [comment]
Marlin 0.9.3 is integrated in jetbrains JRE, and in openjfx 11 & openjdk12.
If you could tell me which extra tests (not in openjdk test suite) do you use ?

**Comment by bourgesl** [comment]
Closing to free my mind
